### ISSUE TITLE
Create "I own the OER World Map" badge

### ISSUE THREAD

**Issue by trugwaldsaenger** [hbz/oerworldmap#167]
User Story
As a supporter of the OER World Map project I want to express my support to the project by displaying an "I own the OER World Map" badge on social networks and other webpages.

Background
- The concept of badges (http://openbadges.org/)  has been discussed frequently at OER conferences
- Pawlowski and Pirkkalainen argue that emotional ownership is important for the adoption of OER`s. Therefore people must be involved in the development process quite early - in the best case before an OER has been developed (http://project.idea-space.eu/results/)
- the RFP agues that the world map should be owned by the community.

**Comment by acka47** [comment]
Given the usage of the verb own (see e.g. http://www.urbandictionary.com/define.php?term=own&defid=987316) I would definitely use another wording for such a button.

**Comment by trugwaldsaenger** [comment]
Yeah - thats kind of funny! What about "I`m an owner of the OER World Map"?

**Comment by trugwaldsaenger** [comment]
Other alternatives might be: "Je suis le OER World Map" or "OER World Map - it's mine!"

**Comment by literarymachine** [comment]
Given the connotation of "Je suis ..." due to quite recent events, I am strictly against this. Actually, I don't like any of this "being" or "owning" the OER world map. How about "I map the world of OER" or something like that?

**Comment by philboeselager** [comment]
In order to use the catchy strength of rhyming, how about something like "We are OER" or "All we are OER" or something similar?
